webstart.ts

```diff
@@ -18,10 +21,25 @@ function stringify(typ: Type, arg: any) : string {
 
 function print(typ: Type, arg : number) : any {
   console.log("Logging from WASM: ", arg);
-  const elt = document.createElement("pre");
+  if (typ.tag == "str") {
```

Comment:
We changed the way to print a string. At the beginning of printing a string, we will input ascii code 256 as the symble. Then the print function will create a <pre> label and store this element in <output> lable. Then, every time we load a ascii code from the memory, this function will convert the ascii code to a character. Get the last child of <output>, which is the place where we store the information of string at front end. In this way, we can wrap a string with only one <pre> lable and we don't need print_enter to change the line anymore, because this label will take such effect.